superdesk/publish/formatters/ninjs_formatter.py

```diff
@@ -62,8 +62,11 @@ def format(self, article, subscriber):
                 if article.get(copy_property) is not None:
                     ninjs[copy_property] = article[copy_property]
 
-            if 'description' in article:
-                ninjs['description_text'] = article['description']
+            if article.get('body_html'):
+                ninjs['body_html'] = self.append_body_footer(article)
+
+            if article.get('description'):
+                ninjs['description_text'] = self.append_body_footer(article)
```

Comment:
Didn't go through the spec. Will change `description_text` to `description_html`. Hope this won't affect Content API!!!